Problem6.hs

```diff
@@ -0,0 +1,18 @@
+import Data.Char
+mytoLower :: Char -> Char
+mytoLower ch
+    | ord 'A' <= ord ch && ord 'Z' >= ord ch  = chr (ord ch + ord 'a' - ord 'A')

```

Comment:
Look for a function with type signature `Char -> Char` in https://hoogle.haskell.org/, it might give you some ideas ;)